katello-installer/katello-installer-base.spec

```diff
@@ -46,32 +45,27 @@ A set of tools for installation of Katello and a the default Capsule.
 %{_sbindir}/capsule-certs-generate
 %{_sbindir}/katello-certs-check
 
-%package -n capsule-installer
-Summary:   Puppet-based installer for a Katello Capsule
+%package -n foreman-installer-capsule
```

Comment:
Updated, new scratch build is http://koji.katello.org/koji/taskinfo?taskID=413218